src/main/java/tds/content/configuration/ContentServiceApplicationConfiguration.java

```diff
@@ -0,0 +1,70 @@
+/***************************************************************************************************
+ * Copyright 2017 Regents of the University of California. Licensed under the Educational
+ * Community License, Version 2.0 (the “license”); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the license at
+ *
+ * https://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required under applicable law or agreed to in writing, software distributed under the
+ * License is distributed in an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for specific language governing permissions
+ * and limitations under the license.
+ **************************************************************************************************/
+
+package tds.content.configuration;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+
+import tds.common.configuration.CacheConfiguration;
+import tds.common.configuration.EventLoggerConfiguration;
+import tds.common.configuration.JacksonObjectMapperConfiguration;
+import tds.common.configuration.RedisClusterConfiguration;
+import tds.common.configuration.SecurityConfiguration;
+import tds.common.web.advice.ExceptionAdvice;
+import tds.itemrenderer.data.xml.itemrelease.Itemrelease;
+
+@Configuration
+@Import({
+    ExceptionAdvice.class,
+    JacksonObjectMapperConfiguration.class,
+    RedisClusterConfiguration.class,
+    CacheConfiguration.class,
+    SecurityConfiguration.class,
+    EventLoggerConfiguration.class,
+})
+public class ContentServiceApplicationConfiguration {
+    @Bean
+    public JAXBContext jaxbContext() throws JAXBException {
```

Comment:
Just fyi, in a Spring app you can do this by simply setting a property in your application.yml or application.properties file.